package.json

```diff
@@ -40,17 +41,16 @@
     "changeset": "npx -y @changesets/cli",
     "release": "pnpm run build --filter={./packages/*}... && pnpm changeset publish"
   },
-  "dependencies": {},
   "devDependencies": {
     "chokidar-cli": "^3.0.0",
     "concurrently": "^8.2.2",
-    "husky": "^9.0.10",
+    "configs": "workspace:*",
+    "husky": "^9.0.11",
     "lint-staged": "^15.2.2",
-    "prettier": "~3.1.0",
+    "prettier": "~3.1.1",
```

Comment:
No dependency changes should be necessary for your PR. Updating `prettier` in particular is problematic as it can result in unrelated formatting changes throughout the repo.

Reply:
this was unintentional. reverted dependencies back to main